app/src/main/java/com/asfoundation/wallet/di/RepositoryModule.kt

```diff
@@ -365,10 +372,29 @@ class RepositoryModule {
   @Singleton
   @Provides
   fun providesWithdrawUseCase(
-    ewt: EwtAuthenticatorService,
-    withdrawRepository: WithdrawRepository
+      ewt: EwtAuthenticatorService,
+      withdrawRepository: WithdrawRepository
   ): WithdrawFiatUseCase {
     return WithdrawFiatUseCase(ewt, withdrawRepository)
   }
 
-}
+  @Singleton
+  @Provides
+  fun providesFiatCurrenciesRepository(@Named("default") client: OkHttpClient,
+                                       objectMapper: ObjectMapper,
+                                       sharedPreferences: SharedPreferences,
+                                       roomFiatCurrenciesPersistence: RoomFiatCurrenciesPersistence,
+                                       conversionService: LocalCurrencyConversionService): FiatCurrenciesRepository {
+    val baseUrl = FiatCurrenciesRepository.CONVERSION_HOST
```

Comment:
CONVERSION_HOST doesn't really need to exist, you can just use `BuildConfig.BASE_HOST` directly